projects/storefrontstyles/scss/cxbase/blocks/forms.scss

```diff
@@ -11,47 +11,55 @@
   }
 }
 
-input.ng-invalid {
-  &.ng-dirty,
-  &.ng-touched {
-    border-color: red;
-  }
-}
+cx-form-errors {
+  display: none;
 
-.form-error {
-  margin: var(--cx-margin, 6px 0);
-  padding: var(--cx-padding, 0 0 0 25px);
-  position: relative;
-  word-break: break-word;
+  p {
+    margin: var(--cx-margin, 6px 0);
```

Comment:
we decided last year to drop any no-theming css variables. There are a few left overs, but generally speaking we tend to not use them for now. Any reason they are added here?

Reply:
no reason, just an old habit I guess. removed